TwL updates PR (#8744)
DataDog/documentation: content/en/tracing/_index.md

</br>

Datadog APM & Distributed Tracing gives deep visibility into your applications with **out-of-the-box performance dashboards** for web services, queues and databases to monitor requests, errors, and latency. Distributed traces **seamlessly correlate** to browser sessions, logs, synthetic checks, network, processes and infrastructure metrics across hosts, containers, proxies, and serverless functions. **Ingest 100% of your traces with no sampling**, search and analyze them live for the last 15-minutes, and use tag-based retention filters to keep traces that matter for your business context for 15-days.

#### Tracing Without Limits: Journey of a Trace

{{< img src="tracing/live_search_and_analytics/tracing_without_limits_lifecycle-0.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="Trace Journey" >}}

> Traces start in your instrumented applications and flow in to Datadog where we ingest 100% of traces up to 50 Traces per second (per APM Host). If needed for high-throughput services, you can view and control the experience using [Ingestion Controls][1].  All ingested traces are available for live search and analytics for 15 minutes and you can use custom tag-based [retention filters][2] to keep exactly the traces that matter for your business context for 15 days search and analytics.

Reviewer: ```suggestion
Traces start in your instrumented applications and flow into Datadog where we ingest 100% of traces up to 50 Traces per second (per APM Host). If you have particularly high-throughput services, you can view and control the experience using [Ingestion Controls][1]. All ingested traces are available for live search and analytics for 15 minutes. Set up custom tag-based [retention filters][2] to keep only the traces that matter for your business context, for 15 days, for search and analytics.
```

### Service Map

[Understand service dependencies][4] with an auto-generated service map from your traces alongside service performance metrics and monitor alert statuses.

{{< img src="tracing/index/ServiceMapInspect.gif" alt="Service Map"  style="width:100%;">}}

### Service Performance Dashboards

[Monitor Service metrics][5] for requests, errors and latency percentiles.  Drill down into database queries or endpoints correlated with infrastructure.

{{< img src="tracing/index/ServicePage.gif" alt="Service Pages"  style="width:100%;">}}

### Live Search

> [Search 100% of your traces][6] by any tag live with no sampling for 15 minutes.

Reviewer: ```suggestion
[Search 100% of your traces][6] by any tag, live with no sampling, for 15 minutes.
```

{{< img src="tracing/live_search/livesearchmain2.gif" alt="Live Search" >}}

### Live Analytics

[Analyze performance by any tag on any span][7] live for 15 minutes during an outage to identify impacted users or transactions.

{{< img src="tracing/live_search/LiveAnalytics2.gif" alt="Live Analytics" >}}

### Trace Retention and Ingestion

[Retain the traces that matter most to you][8] with tag-based retention filters and perform analytics on all indexed spans for 15 days.

{{< img src="tracing/index/RetentionFilterTracingPage.png" alt="Trace Retention and Ingestion"  style="width:100%;">}}

### Deployment Tracking

[Monitor service performance][9] and compare between versions for rolling, blue/green, shadow, or canary deployments.

> {{< img src="tracing/deployment_tracking/VersionComparison.png" alt="Versions on the Service Page"  style="width:100%;">}}

Reviewer: Did you mean to change the deployment tracking png name here?

Author: Yeah - trying out a different image - have a look and see if you think it's more compelling than what we have on prod
